bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Control.java

```diff
@@ -3688,7 +3688,7 @@ public void setRedraw (boolean redraw) {
 private boolean embedsWin32Control () {
 	if (this instanceof Browser) {
 		// The Edge browser embeds webView2
-		return (getStyle() & SWT.EDGE) != 0;
+		return (getStyle() & SWT.IE) == 0;
```

Comment:
Now that we always have the "EDGE" flag set, I think we should keep this check as it was before

Reply:
We do not always have the flag set. By default, the style flag will simply be `SWT.NONE`.

Reply:
In its current state (075d949dc7e5f3c27e68075e75a9126c82913767) this wouldn't work because `SWT.EDGE` is set `if (current.equalsIgnoreCase ("edge") && "win32".equals (platform))` which means one has to explicitly provide the parameter. Not providing any parameter still sets the style to `SWT.NONE`

Reply:
Sounds good. I've adapted the code with c0466d7.